Doc/update (#351)
vtex-apps/product-summary: docs/README.md

| Block name     | Description | 
| -------------- | ----------- | 

> | [`list-context.product-list`](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-summary-productsummarylist) | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Renders the list of products in the Product Summary component. It is responsible for fetching product data and providing it to the `product-summary.shelf` block. The last one, in turn, provides the product data to its children blocks. | 

Reviewer: ```suggestion
| [`list-context.product-list`](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-summary-productsummarylist) | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Renders the list of products in the Product Summary component. It fetches product data and provides it to the `product-summary.shelf` block. This block, then, provides its child blocks with the product data. | 
```

| `product-summary.shelf` | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Logical block responsible for providing the needed structure for the Product Summary component through its child blocks (listed below). 
| [`product-summary-attachment-list`](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-summary-productsummaryattachmentlist) | Renders a list for the product [attachments](https://help.vtex.com/tutorial/adding-an-attachment--7zHMUpuoQE4cAskqEUWScU). | 
| [`product-summary-brand`](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-summary-productsummarybrand)         | Renders the product brand. | 

> | [`product-summary-buy-button`](https://vtex.io/docs/components/all/vtex.product-summary/product-summary-buy-button) | Renders the Buy Button. Notice that this block should only be configured if your store still uses the [Minicart v1](https://github.com/vtex-apps/minicart/blob/383d7bbd3295f06d1b5854a0add561a872e1515c/docs/README.md). When using the [Minicart v2](https://developers.vtex.com/vtex-developer-docs/docs/vtex-minicart), you should configure the [**Add To Cart Button**](https://developers.vtex.com/vtex-developer-docs/docs/vtex-add-to-cart-button) instead.  | 

Reviewer: ```suggestion
| [`product-summary-buy-button`](https://vtex.io/docs/components/all/vtex.product-summary/product-summary-buy-button) | Renders the Buy Button. This block must be configured only if your store uses the [Minicart v1](https://github.com/vtex-apps/minicart/blob/383d7bbd3295f06d1b5854a0add561a872e1515c/docs/README.md). If your store uses the [Minicart v2](https://developers.vtex.com/vtex-developer-docs/docs/vtex-minicart), please configure the [**Add To Cart Button**](https://developers.vtex.com/vtex-developer-docs/docs/vtex-add-to-cart-button) instead.  | 
```

| [`product-summary-description`](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-summary-productsummarydescription) | Renders the product description. | 
| [`product-summary-image`](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-summary-productsummaryimage) | Renders the product image. | 
| [`product-summary-name`](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-summary-productsummaryname) | Renders the product name. | 
| [`product-summary-sku-name`](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-summary-productsummaryskuname) | Renders the selected sku name. | 

> | `product-summary-price` | ![https://img.shields.io/badge/-Deprecated-red](https://img.shields.io/badge/-Deprecated-red) The Product Summary Price block, responsible for rendering the product price, has been deprecated in favor of the [Product Price](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-price) app. Although support for this block is still granted, we strongly recommend you to use the Product Price app's blocks instead. | 

Reviewer: ```suggestion
| `product-summary-price` | ![https://img.shields.io/badge/-Deprecated-red](https://img.shields.io/badge/-Deprecated-red) Renders the product price. This block has been deprecated in favor of the [Product Price](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-price) app. Although support for this block is still available, we strongly recommend that you use the Product Price app. | 
```

| [`product-summary-sku-selector`](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-summary-productsummaryskuselector) | Renders the SKU Selector block. | 
| [`product-specification-badges`](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-summary-productsummaryspecificationbadges) | Renders badges based on the product specifications. |

> 2. Add the `list-context.product-list` block to the store theme template where you desire to display a product list and declare the `product-summary.shelf` in its block list. For example:

Reviewer: ```suggestion
2. Add the `list-context.product-list` block to a store template of your choice and declare the `product-summary.shelf` in its block list. For example:
```

  },
```

> ℹ️ Info 
> 

> > Notice that although the `product-summary.shelf` refers to the shelf component in its name, the block does not need to be necessarily used  when building a Shelf component. It can and should be used whenever you want to render summarizing product information in other components, such as the [Minicart](https://vtex.io/docs/components/all/vtex.minicart/) and those found on the [Search Results](https://vtex.io/docs/components/all/vtex.search-result/) page.

Reviewer: ```suggestion
> Although the name of the block 'product-summary.shelf' alludes to the Shelf component, this block is not required to create a Shelf component. The Product Summary Shelf is used to present summary product information in other components, such as the [Minicart](https://developers.vtex.com/vtex-developer-docs/docs/vtex-minicart) and the [Search Results](https://developers.vtex.com/vtex-developer-docs/docs/vtex-search-result) page.
```

> 3. Based on the product information you desire to have rendered in the product list, choose which blocks from the exported list above will be sent as the `product-summary.shelf` children. In a scenario in which we want to display the product name, description, image, price, a SKU selector, and then a Buy Button, it would go as follows:

Reviewer: ```suggestion
3. Add the blocks from the list above as children of the `product-summary.shelf`, considering the product information you want to present in the product list. Take the following example in which the product name, description, image, price, SKU selector, and Buy Button are all displayed in the Product Summary:
```

}
```

> > ℹ️ Info
> >
> > In order to configure and better understand each of the Product Summary exported blocks, go through their respective documentation.

Reviewer: ```suggestion
```
